core/client/hammer/benchmark.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Could do with a better filename -- this isn't entirely benchmark-specific code.

Reply:
hmm. well it was until we removed the benchmarking bit :-)